### PR TITLE
SAK-41081 Move Rubrics CSS/SCSS styles to library

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+            width: 100%;
+            height: 100%;
+            display: table;
+            border: 1px solid #d3d3d3;
+            border-width: 1px 0 0 0;
+            &.sortable-ghost, .sortable-ghost {
+                opacity: 0.25;
+            }
+            &.sortable-drag, .sortable-drag {
+                cursor: grabbing;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: I prefer to put the non-browser-prefixed property last so the certified property gets applied (last) as browsers adopt the new property going forward.

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+                cursor: -webkit-grabbing;
+                cursor: -moz-grabbing;
+            }
+            .criterion-detail {
+                position: relative;
+                padding: 10px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: There is a variable called `$standard-spacing` that could be used here.

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+                cursor: -moz-grab;
+                &:active {
+                    cursor: grabbing;
+                    cursor: -webkit-grabbing;
+                    cursor: -moz-grabbing;
+                }
+            }
+            .criterion-ratings {
+                .rating-item {
+                    sakai-rubric-criterion-rating-edit {
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: A few cases of potential class missing the starting period. (Unless it is a custom element)

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+                opacity: 0.5;
+                position: absolute;
+                padding-top: 5px;
+                right: -10px;
+                top: 0;
+                bottom: 0;
+                width: 20px;
+                z-index: 5;
+                &:hover {
+                    opacity: 1;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: Why do you change the opacity if the display is changed too?

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+                }
+            }
+
+            &:hover, &.active {
+                .add-criterion-item {
+                    display: block;
+                }
+                sakai-rubric-criterion-edit {
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: A few cases of potential class missing the starting period. (Unless it is a custom element)

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 6]
```diff
@@ -0,0 +1,415 @@
...
+                    }
+                    .points {
+                        color: #444444;
+                        font-style: italic;
+                    }
+                }
+            }
+
+            .criterion-actions {
+                padding: 10px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: There is a variable called `$standard-spacing` that could be used here.

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 7]
```diff
@@ -0,0 +1,415 @@
...
+            &.criterion-item-title {
+                font-weight: normal;
+                color: #3a3a3a;
+            }
+        }
+
+        &.grading {
+            .rating-item {
+                cursor: default;
+                &:hover:not(.student) {
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: Should ":focus" also be used here?

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 8]
```diff
@@ -0,0 +1,415 @@
...
+            }
+        }
+    }
+}
+
+.rubric-totals {
+    &.sakai-rubric-criterion-grading,
+    &.sakai-rubric-criterion-student {
+        text-align: right;
+        padding: 5px 10px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: There is a variable called `$standard-spacing` that could be used here.

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 9]
```diff
@@ -0,0 +1,415 @@
...
+    }
+}
+
+sakai-rubric-grading-comment,
+sakai-rubric-student-comment {
+    iron-icon.comment-icon {
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: Another case of "iron-icon" without a period.

File: library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss [hunk 10]
```diff
@@ -0,0 +1,415 @@
...
+        justify-content: flex-end;
+        .criterion-title{
+            margin-right: auto;
+            margin-bottom: inherit;
+            align-self: center;
+            font-size: 14px;
+        }
+        .buttons{
+            align-self: center;
+            margin-left: 10px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: There is a variable called `$standard-spacing` that could be used here.

Comment (josecebe) [reply library/src/morpheus-master/sass/modules/tool/rubrics/_rubrics.scss]: Everything done, thanks for the suggestions @fostersdesign !

**Comment by mpellicer** [comment]
@josecebe would you mind to resolve the conflicts? There are conflicts after merging #6390

**Comment by mpellicer** [comment]
The conflicts have been resolved, awaiting for the build to merge it.

**Comment by fostersdesign** [comment]
Thanks for making the changes, @josecebe.

However, I wish I had had time to re-review your changes again today before it was merged: you shouldn't have put the variables in the Bootstrap variables files. They should have been in a separate variables file in the morpheus-master/sass/modules/tool/rubrics/ directory instead. 

I wish more time was given by the Sakai Core team to review these PRs by more people before they are merged.

**Comment by mpellicer** [comment]
hi @fostersdesign , this PR is very critical because needs to be merged just after #6390, if not Rubrics will be broken for QA. 

Both PR resolved a big important problem for the future of Rubrics, moving the CSS to Morpheus (Library) and remove all the local JS files in favor of Webjars sounds very very well.

We'll be happy to make more corrections in future PRs, we don't want to break Rubrics waiting for this PR.

Miguel

**Comment by fostersdesign** [comment]
I have created #6432 to move these SASS variables to the correct spot.

**Pull request merged**